strings/tips.txt

```diff
@@ -125,9 +153,11 @@ As a Botanist, you can mutate the plants growing in your hydroponics trays with
 As a Botanist, you should look into increasing the potency of your plants. This increases the size, amount of chemicals, points gained from grinding them in the biogenerator, and lets people know you are a proficient botanist.
 As a Botanist, you can combine production trait chemicals just like a Chemist. Chlorine (blumpkin) + radium and phosphorus (glowshrooms) equals unstable mutagen!
 As a Cook, you can load your food into snack vending machines.
+As a Cook, you can create a very wide variety of food with the crafting menu (that little T symbol next to your intents).
```

Comment:
It's supposed to be a hammer. Everyone thinks of it as a t though, because it really looks like one.